Dockerfile

```diff
@@ -12,20 +12,22 @@ RUN apt-get update -qqy && DEBIAN_FRONTEND=noninteractive apt-get install -qqy -
     libqt5gui5 \
     libqt5widgets5 \
     libqt5printsupport5 \
-    libicu57 \
-    libmpfr4 \
+    libicu63 \
+    libmpfr6 \
 && rm -rf /var/lib/apt/lists/* /var/cache/apt/*
 
-ARG VERSION=2.1.7
-ARG FILE=MiniZincIDE-$VERSION-bundle-linux-x86_64
+ARG VERSION=2.3.1
+ARG FILE=MiniZincIDE-$VERSION-bundle-linux
```

Comment:
This argument could be renamed to `NAME` for example. Would probably be more clear in the RUN part.

Reply:
FILE -> NAME